metatron-app/metatron-discovery
# Change how the time filter specifies the last date

### Description
Change how the time filter specifies the last date

**Related Issue** : 


### How Has This Been Tested?
1. Add a time filter.
2. Select Last Time to verify that the time is displayed as Last Data Time.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
